packages/xo-web/src/xo-app/home/index.js

```diff
@@ -705,13 +708,18 @@ export default class Home extends Component {
       createFilter(() => this.props.items, this._getFilterFunction),
       () => this.props.location.query.backup,
       () => this.props.jobs,
-      (filteredItems, backup, jobs) => {
+      () => this.props.schedulesByJob,
+      (filteredItems, backup, jobs, schedulesByJob) => {
         if (backup === undefined) {
           return filteredItems
         }
-
         const backedUpVms = uniq(
-          flatMap(jobs, job => filter(filteredItems, createPredicate(omit(job.vms, 'power_state'))))
+          flatMap(jobs, job =>
+            filter(
+              filteredItems,
+              item => createPredicate(omit(job.vms, 'power_state'))(item) && schedulesByJob[job.id][0].enabled
```

Comment:
Probably much faster to filter the jobs first.

Reply:
Also be careful, because with your change, you're recreating the predicate for each item.